tests/integration_tests/cachekeys/api_tests.py

```diff
@@ -149,9 +149,9 @@ def test_invalidate_existing_caches(invalidate):
     )
 
     assert rv.status_code == 201
-    assert cache_manager.cache.get("cache_key1") is None
-    assert cache_manager.cache.get("cache_key2") is None
-    assert cache_manager.cache.get("cache_key4") is None
+    assert cache_manager.cache.get("cache_key1") == "value"
+    assert cache_manager.cache.get("cache_key2") == "value"
+    assert cache_manager.cache.get("cache_key4") == "value"
```

Comment:
@dpgaspar @etr2460 @bkyryliuk  @Antonio-RiveroMartnez @Yoann-tildev Do you know why this `cache_manager` behavior has changed in new version of `flask-caching`?

Reply:
no idea, have you debugged it

Reply:
Hi @bkyryliuk! I see you created this unit test. Is this change critical for superset?